tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "lib": [
       "dom",
       "es2015"
-    ]
+    ],
+    "skipLibCheck": true
```

Comment:
Do we need this if we also bump to TS 3.1.4?

Reply:
Yes otherwise the `yarn problems` command fails with: 

```
node_modules/rollup/dist/typings/watch/index.d.ts:4:30 - error TS2307: Cannot find module 'chokidar'.

4 import { WatchOptions } from 'chokidar';
                               ~~~~~~~~~~
```

Reply:
I just pushed another update to fix those problems without requiring `skipLibCheck` (basically there was an issue with the way rollup < 0.58 includes its own type definitions).